modules/persistent-storage-csi-cloning-overview.adoc

```diff
@@ -3,19 +3,27 @@
 // * storage/container_storage_interface/persistent-storage-csi-cloning.adoc
 
 [id="persistent-storage-csi-cloning-overview_{context}"]
-= Overview of CSI volume cloning
+= How CSI volume cloning works
 
-A Container Storage Interface (CSI) volume clone represents a duplicate of an existing persistent volume that is its own unique volume on the system at a particular point in time. With cloning, the back-end device creates an exact duplicate of the specified volume, rather than creating a new empty volume. After it has been dynamically provisioned, a volume clone can be consumed as any standard volume would be.
+Cloning creates an exact duplicate of the specified volume on the back-end device, rather than creating a new empty volume. After dynamic provisioning, you can use a volume clone just as you would use any standard volume.
 
-Volume cloning is similar to volume snapshots, although it offers a more efficient method for creating a pre-populated volume copy. For example, a cluster administrator can duplicate a cluster volume by creating another instance of the existing cluster volume.
+No new API objects are required for cloning. The existing `dataSource` field in the PersistentVolumeClaim object is expanded so that it can accept the name of an existing PersistentVolumeClaim in the same namespace.
 
-There are no new API objects required for cloning. The existing `dataSource` field in the PersistentVolumeClaim object is expanded to be able to accept the name of an existing PersistentVolumeClaim in the same namespace.
+== Definition of CSI volume cloning
```

Comment:
So this definitely seems odd to me. You've described how CSI volume cloning works first, and then give a definition of it here? Seems backwards, but also I'm not a fan of the "Definition of" heading.

Reply:
Agree! It was suggested that I break this out and use those subheads but your reaction makes sense. I'm going to revert to "Overview" and change the sentence order again slightly.